### DISCUSSION TITLE
Option to disable notification when there is a connection with permanent password and view mode

### DISCUSSION THREAD

**Discussion by Bruce-gits** [rustdesk/rustdesk#4612] [Feature Request]
I am using this software to monitor 5 stations for work computers, I am the person that does the audits to my co-workers, our stats shows that sales have dropped and work efficiency because they are starting to use work computers for personal matters since we switch to Rustdesk because each time we do an audit , they know when we access the work stations. WE DO NOT allow work computers for personal matters and hence for doing this , they get warning and second time will be fired. When people know , that the computers are being monitored, they are discourage from doing/ misused of computer. THERE SHOULD BE AN ADMIN OPTION TO TURN OFF NOTIFICATION when there is a connection on a password. Would it be kind for the developers to consider this option please?

Thanks

**Comment by deep-soft** [comment]
https://github.com/rustdesk/rustdesk/discussions/4424#discussioncomment-6142576
